tests/helpers.py

```diff
@@ -83,6 +84,7 @@
         b"T\xfd,\xf1I\x00\x00\x00^\xad\x00\x00<\x00\x05\xe0",
         b"T\xfd,\xf1I\x00\x00\x00^\xad\x00\x00<\x00\x06\xe0",
         b"Q\xfd,\xf1I\x00\x00\x00^\xad\x00\x00<\x00\x07\xe0",
+        b"j!\xcf\x14D\xce\xe6B\xab\xe0\xd9\xbey\x0cMK",
```

Comment:
Can you double check if pymssql returns a bytes object instead of a UUID object?